content/rc/security/access-control/multi-factor-authentication.md

```diff
@@ -2,7 +2,7 @@
 Title: Multi-factor authentication
```

Comment:
@cmilesb do we maybe want to add to a note in this section (similar to the MFA section in SAML)?

>  If MFA enforcement is enabled, note that Redis Cloud does not enforce MFA for SAML users since the identity provider handles MFA management and enforcement.